### PR TITLE
NOISSUE - Rename path variables in InfluxDB wirter

### PR DESCRIPTION
**Pull request by dborovcanin**
Signed-off-by: Dušan Borovčanin <dusan.borovcanin@mainflux.com>

### CODE REVIEW

File: cmd/influxdb/main.go [hunk 1]
```diff
@@ -12,8 +12,8 @@ import (
 	"github.com/mainflux/mainflux"
 	log "github.com/mainflux/mainflux/logger"
 	"github.com/mainflux/mainflux/writers"
-	influxdb "github.com/mainflux/mainflux/writers/influxdb"
-	nats "github.com/nats-io/go-nats"
+	"github.com/mainflux/mainflux/writers/influxdb"
+	"github.com/nats-io/go-nats"
```
Comment (anovakovic01) [inline cmd/influxdb/main.go]: Please return `nats` alias. I think that improves readability (You don't need to open files from `github.com/nats-io/go-nats` to see package name).

File: writers/influxdb/README.md [hunk 2]
```diff
@@ -39,8 +39,8 @@ make install
```
Comment (anovakovic01) [inline writers/influxdb/README.md:59]: Add run command, same as in other `README` files.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by nmarcetic — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**